### ISSUE TITLE
bug: info (and lower) level logs are not printed because logging level is not being set

### ISSUE THREAD

**Issue by HarikrishnanBalagopal** [foundation-model-stack/fms-hf-tuning#243]
This never gets printed even with `--log_level=debug`
https://github.com/foundation-model-stack/fms-hf-tuning/blob/9e4b49f25a7d9a383cd8e5dc1e3175e09197f890/tuning/sft_trainer.py#L255

because the logging level is not set here https://github.com/foundation-model-stack/fms-hf-tuning/blob/9e4b49f25a7d9a383cd8e5dc1e3175e09197f890/tuning/sft_trainer.py#L112

https://huggingface.co/docs/transformers/main/en/main_classes/trainer#transformers.TrainingArguments.log_level

**Comment by Ssukriti** [comment]
@anhuong for keeping track. we can assign to someone in next 2 weeks while doing QLoRA testing in parallel

**Comment by kmehant** [comment]
related and possibly duplicate of - https://github.com/foundation-model-stack/fms-hf-tuning/issues/195

**Comment by Abhishek-TAMU** [comment]
This [PR](https://github.com/foundation-model-stack/fms-hf-tuning/pull/270) fixes the issue. Thanks!
